### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by Abdelaziz-Ali**
### General requirements
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config). Remember that this is your first React project and it requires a special linters setup. Check the instructions in the linters-config README.
- Make sure that you used correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

**Project requirements**
- First, make sure you have [Node.js](https://nodejs.org/en/) installed on your machine
- The easiest way to create a React application is using [Create React App](https://create-react-app.dev/)
- The application should run in the browser without errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
+    "@vitejs/plugin-react": "^4.0.0",
+    "eslint": "^8.38.0",
+    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-react-refresh": "^0.3.4",
+    "vite": "^4.3.2"
```
Comment (Olamarx) [inline package.json:24]: - [ ] While you did a beautiful job, it is required you use create-react-app and not vite, kindly prioritize the usage of React library.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Abdelaziz-Ali, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file professional ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (Abdelaziz-Ali) [reply package.json:24]: Hello, @Olamarx
  I thought they said it would be faster not required, What do you think?

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Abdelaziz-Ali 👋

Great work implementing the changes suggested by the reviewer 💪

**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ `create-react-app` is used
-  ✅ Great work using the right Git Flow 
-  ✅ Your README looks professional, well done!
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**